amundsen_application/api/utils/notification_utils.py

```diff
@@ -0,0 +1,37 @@
+from flask import render_template
+from typing import Dict
+
+
+def get_notification_content(notification_type: str, data: Dict) -> Dict:
+    """
+    Forms a short version of a table Dict, with selected fields and an added 'key'
+    :param table: Dict of partial table object
+    :return: partial table Dict
+
+    TODO - Unify data format returned by search and metadata.
+    """
+    notification_type_dict = {
```

Comment:
Let's protect ourselves from a bug that would happen if we try to access `notification_type_dict` with an invalid `notification_type`. We want to make sure we don't try to access the `'html'` key on an undefined variable. There's a couple ways to do this, you could write a helper method, or you could use `.get` and return a default value. i.e. `notification_type_dict.get(notification_type, {}).get('html')`

The second argument of `.get()` on python dictionaries is what will get returned if the value doesn't exist, else it  returns `None`.